src/app/space-wizard/components/forge-app-generator/forge-app-generator.component.scss

```diff
@@ -20,17 +20,17 @@
 }
 
 .btn-left-multi-select-button {
-  min-width: unset;
+  //min-width: unset;
   border-left-style: none;
   border-top-style: none;
-  border-bottom: solid 2px #efefef;
+  border-bottom: solid 3px #efefef;
```

Comment:
@mindreeper2420 please make an issue for this, since it won't get fixed this week.